lib/device_map.py

```diff
@@ -171,6 +171,16 @@
                        'fastboot': False,
                        'mach': 'mvebu'}
 
+armada_8040_db = {'device_type': 'armada-8040-db',
+                       'templates': ['generic-arm64-dtb-kernel-ci-boot-template.json'],
+                       'kernel_defconfig_blacklist': [],
+                       'defconfig_blacklist': ['arm64-allmodconfig'],
+                       'kernel_blacklist': ['v4.6'],
```

Comment:
Hi Florent,

The Armada 8040 DB DTS has only been merged in 4.9 apparently so this kernel version blacklisting is useless.